extensions/interactions/LogicProof/static/js/README.txt

```diff
@@ -23,11 +24,12 @@ The code is contained in the following files:
   - conversion.ts. Used to convert symbols into unicode logic symbols as users
  type.
 
-In order to update generatedDefaultData.js:
+In order to update generatedDefaultData.ts:
 1. Make the desired changes in tools/strings.ts.
 2. Open tools/demonstration.html and check everything has succeeded.
 3. Click "Request javascript" at the bottom of the page.
-4. Paste the result into generatedDefaultData.js.
+4. Paste the result into generatedDefaultData.js (
+   Below the import statement and above the export statement).
```

Comment:
Below --> below

Also, move the first few words of this line to the end of the previous line.

Reply:
Done